Automate_EDA_with_DataExplorer.Rmd

```diff
@@ -0,0 +1,174 @@
+# Automate EDA with DataExplorer
```

Comment:
Both the .rmd file name and the folder file name in the resources should be all lower case:

See 17.2.2 for filename requirements:

https://jtr13.github.io/cc20/tutorial-for-pull-request-mergers.html